README.md

```diff
@@ -53,6 +53,10 @@ selecting the `Use DiCy` option in the settings page. [DiCy][] will be installed
 automatically and so no further action is required for either TeX Live or
 MiKTeX.
 
+### Code Highlighting
```

Comment:
Rename this to "Syntax Highlighting" to better match the terminology used within Atom.